cmd/runtimecfg/vr-ids.go

```diff
@@ -34,7 +34,7 @@ func runVRIds(cmd *cobra.Command, args []string) error {
 	clusterType := v.Type()
 	for i := 0; i < v.NumField(); i++ {
 		fieldName := clusterType.Field(i).Name
-		if strings.HasSuffix(fieldName, "RouterID") {
+		if strings.HasSuffix(fieldName, "RouterID") && fieldName != "DNSVirtualRouterID" {
```

Comment:
Shouldn't we just remove the DNS VIP fields from the config struct? They haven't been used in several releases now.